platforms/jvm/language-jvm/src/main/java/org/gradle/api/tasks/compile/BaseForkOptions.java

```diff
@@ -26,6 +26,8 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
+import static org.apache.commons.lang3.StringUtils.isBlank;
```

Comment:
💅 Nitpick, I don't think we tend to use static imports in this codebase -- this might be confusing.

Reply:
Removed static import.